cockpit/leapp.html

```diff
@@ -803,63 +866,65 @@ <h3 class="pull-left">
         }
 
         function load_target_container_names() {
-            var ctx = new_leapp_call(["check-target", "-t", "127.0.0.1"]);
+            var cmd = ["check-target", {
+                target_host: "localhost"
```

Comment:
sometimes you're using localhost and sometimes 127.0.0.1, can you use one value?